6_extend_catalog.ipynb

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    url
</pre><p>Will this print out the URL for the student to see it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheapandfair/cheapandfair-gateways-2024/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>yes</p>